docs/DeveloperGuide.md

```diff
@@ -307,31 +310,112 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `ArtBuddy`
+and the **Actor** is the `user`, unless specified otherwise)
+
+**Use case: Delete a customer**
+
+**MSS**
+
+1. ArtBuddy shows a list of customers
+2. User requests to delete a specific customer in the list
+3. ArtBuddy deletes the customer and all the customer's commissions
```

Comment:
Should we specify that AB will display the updated customer list after deleting the customer with no customer or another one selected?

Reply:
Ok I will add that in